spec/features/workflow/viewing_a_document_with_no_creator_spec.rb

```diff
@@ -1,12 +1,14 @@
 # frozen_string_literal: true
 
 RSpec.feature "Viewing a document with no creator" do
-  scenario "User finds document without a creator and views it" do
+  scenario "User finds an unedited document without a creator and views it" do
```

Comment:
I think this spec is more 'User views a document without a creator/editor'. We should avoid implying that this is possible within the app, as it only happens for imported stuff, which may have been edited by someone, we just don't know who.

Reply:
Agreed, this makes the test intention clearer - now updated 👍